src/MainView.mc

```diff
@@ -6,7 +6,9 @@ using TextInput;
 
 var _error = "";
 var _errorTicks = 0;
+var _enableAntiAlias = false;
```

Comment:
As these globals are also used by the glance view now (more specifically `_errorTicks`), I'd suggest to move them to the `App.mc`.

Reply:
No need to change though, I can do such refactorings after merging.